subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -501,7 +619,18 @@ uint8_t ll_cis_parameters_test_set(uint8_t cis_id, uint8_t nse,
 {
 	uint8_t cis_idx = ll_iso_setup.cis_idx;
 
-	if (cis_idx >= CONFIG_BT_CTLR_CONN_ISO_STREAMS_PER_GROUP) {
+	if ((cis_id > BT_HCI_CIS_ID_VALID_MAX) ||
+	    ((c_sdu & ~BT_HCI_MAX_SDU_VALID_MASK) != 0) ||
+	    ((p_sdu & ~BT_HCI_MAX_SDU_VALID_MASK) != 0)) {
+		return BT_HCI_ERR_INVALID_PARAM;
+	}
+
+	if ((!c_phy || (c_phy & ~BT_HCI_PHY_VALID_MASK) != 0) ||
+	    (!p_phy || (p_phy & ~BT_HCI_PHY_VALID_MASK) != 0)) {
```

Comment:
```suggestion
	    ((c_sdu & ~BT_HCI_MAX_SDU_VALID_MASK) != 0U) ||
	    ((p_sdu & ~BT_HCI_MAX_SDU_VALID_MASK) != 0U)) {
		return BT_HCI_ERR_INVALID_PARAM;
	}

	if ((!c_phy || (c_phy & ~BT_HCI_PHY_VALID_MASK) != 0U) ||
	    (!p_phy || (p_phy & ~BT_HCI_PHY_VALID_MASK) != 0U)) {
```

Reply:
It seems that these checks are added multiple times; have you considered a common function?

Reply:
A common masking-checker function? E.g. an `IS_VALID_MASK(c_phy, BT_HCI_PHY_VALID_MASK)` macro or even a function?

Reply:
Ex.: 
```
#define IS_VALID_MASK(value, valid_mask) \
	(((value) & ~(valid_mask)) != 0U)
```

Reply:
Reading this again I suspect you refer to the duplicate code between `ll_cis_parameters_set` and `ll_cis_parameters_test_set`. Will make a function for that